assume_account/main.tf

```diff
@@ -0,0 +1,18 @@
+terraform {
+  required_version = ">= 0.14.0"
+
+  required_providers {
+    aws = {
+      source  = "hashicorp/aws"
+      version = "3.37.0"
```

Comment:
Is this meant to be a reusable module? If not, no need to pin here. Just use the `.terraform.lock.hcl`

Reply:
So I don't need to pin the version, but add the `.terraform.lock.hcl` file instead?